tests/e2e/tests/pilot/pilot_test.go

```diff
@@ -328,6 +329,13 @@ func (t *testConfig) Setup() (err error) {
 		}
 	}
 
+	// For multicluster tests, add the remote cluster into the mesh
+	// and verify the multicluster service mesh before starting tests
+	if len(t.Kube.Clusters) > 1 {
+		err = addAndVerifyRemoteCluster()
```

Comment:
Also might want a note to indicate this call ends up triggering the entire install of Istio and the apps on the remote cluster.

Reply:
I was struggling with the names. Actually the install part (both remote istio and the apps) is already done before this call. The call is to register the remote cluster and verify the muticluster mesh. Let me change it to createAndVerifyMCMeshConfig()